documentation/js-sdk-guide/antora.yml

```diff
@@ -1,6 +1,6 @@
 name: js-sdk-guide
 title: JavaScript SDK
-version: master
+version: 2.0.0
```

Comment:
You must take care that `main` has a different version than the tags, i.e., IMO this should stay on `master` on the `main` branch.